io/pci/PowerNVEEH.py.data/PowerNVEEH.yaml

```diff
@@ -0,0 +1,6 @@
+scenario:
+            #max_freeze: 5
+            #function: 4
+            #err: 1
+            #pe: "0001:0c:00.0"
```

Comment:
this can be `pci_device` as in other tests

Reply:
I think pe is right abbreviation here.

Reply:
@vrbagalkote Agreed. But in other tests such as hotplug or driver bind, the parameter name is `pci_device`.

We want all names to be same, so that they can be changed automatically in one shot. So, either:
1. change this to `pci_device`
2. change in all other tests `pci_device` to `pe`